example/provider-local/base/cloudprofile.yaml

```diff
@@ -8,6 +8,7 @@ spec:
   - name: local
   kubernetes:
     versions:
+    - version: 1.23.6
     - version: 1.23.1
```

Comment:
You could drop this line now

Reply:
done

Reply:
Please make the changes in the e2e tests also.